hash/apple2_flop_clcracked.xml

```diff
@@ -41050,7 +41052,8 @@ license:CC0-1.0
 		<description>Mathematics Action Games: Pyramid Puzzler (4am crack)</description>
 		<year>1985</year>
 		<publisher>Mindscape</publisher>
-		<info name="developer" value="Scott, Foresman and Company" />
+		<info name="developer" value="Scott" />
```

Comment:
Looks like single entry

Reply:
"Scott, Foresman and Company" is a single company name (founded in 1896!), not a list of individuals.
See https://en.wikipedia.org/wiki/Scott_Foresman